specification/ai/Azure.AI.Agents/tools/models.tsp

```diff
@@ -782,6 +882,49 @@ model ConnectedAgentDetails {
   description: string;
 }
 
+/**
+ * The details of the browser automation tool.
+ */
+@added(Versions.v2025_05_15_preview)
+model BrowserAutomationDetails {
+  @doc("Information about the remote browser to be used for automation.")
+  connection: BrowserAutomationRemoteBrowserConnection;
+}
+
+/**
+ * The connection ID containing information about the remote browser.
+ */
+@added(Versions.v2025_05_15_preview)
+model BrowserAutomationRemoteBrowserConnection {
+  @doc("The connection ID for the remote browser.")
+  @encodedName("application/json", "id")
+  connectionId: string;
+}
+
+/**
+ * The details of the Deep Research tool.
+ */
+@added(Versions.v2025_05_15_preview)
+model DeepResearchDetails {
+  @doc("The deep research model deployment name.")
+  @encodedName("application/json", "deep_research_model")
+  deepResearchModelDeploymentName: string;
+
+  @doc("The array containing Bing grounding connection IDs to enhance deep research capabilities.")
+  @encodedName("application/json", "bing_grounding_connections")
+  bingGroundingConnections: BingGroundingConnectionInformation[];
+}
+
+/**
+ * The connection resource ID for the Bing grounding resource.
+ */
+@added(Versions.v2025_05_15_preview)
+model BingGroundingConnectionInformation {
```

Comment:
A couple of questions:
- How is this new model connected to BrowserAutomation and DeepResearch?
- What is the purpose of this new model? Are we replacing an existing model with it?

Reply:
Since this is only going to be used to define the Deep Research Tool, and to differentiate it from the other Bing Grounding classes, I suggest calling this TypeSpec model `DeepResearchBingGroundingConnection` (no need for `Information` at the end...) instead of `BingGroundingConnectionInformation`